### PR TITLE
Fix skip to content link position

### PR DESCRIPTION
**Pull request by DenisHdz**
Resolves #NUMBER

**Overall change:**
The skip to content link was in the wrong position on IE:

skip-to-content-bug

**Code changes:**
- Add `top` and `left/right` properties to the `SkipLink`
- Add `Script` link to the last story
- Export `ScriptLink` from a separated folder to reduce `index` length.

---

- [X] I have assigned myself to this PR and the corresponding issues
- [ ] Automated jest tests added (for new features) or updated (for existing features)
- [X] This PR requires manual testing

### CODE REVIEW

**Review by OlgaLyubin — COMMENTED** [review]
For `rtl` it seems like the text from the `ScriptLink` is overflowing the `Skip to content`
Screenshot 2019-11-25 at 11 35 26

**Comment by DenisHdz** [comment]
@OlgaLyubin It would be solved adding z-index to the `SkipLink`, but we have decided to put it on the right side for `rtl` services, so it should be fine.

**Review by OlgaLyubin — APPROVED** [review]
Looks good on IE 👍

File: packages/components/psammead-brand/src/SkipLink/index.jsx [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+  height: 1px;
+  width: 1px;
+  overflow: hidden;
+  padding: ${TOP_BOTTOM_SPACING} ${GEL_SPACING};
+  background-color: #ffffff;
+  border: ${SKIP_LINK_BORDER} solid #000;
+  color: ${SKIP_LINK_COLOR};
+  text-decoration: none;
+  ${({ script }) => script && getPica(script)};
+  ${({ service }) => service && getSansRegular(service)}
```
Comment (tochwill) [inline packages/components/psammead-brand/src/SkipLink/index.jsx]: Missing a ; on the end? `${({ service }) => service && getSansRegular(service)};`

File: packages/components/psammead-brand/src/SkipLink/index.jsx [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+
+const SkipLink = styled.a`
+  position: absolute;
+  clip-path: inset(100%);
+  clip: rect(1px, 1px, 1px, 1px);
+  height: 1px;
+  width: 1px;
+  overflow: hidden;
+  padding: ${TOP_BOTTOM_SPACING} ${GEL_SPACING};
+  background-color: #ffffff;
```
Comment (OlgaLyubin) [inline packages/components/psammead-brand/src/SkipLink/index.jsx]: Might use `${C_WHITE}` for this one

**Review by tochwill — APPROVED** [review]

**Review by Bopchy — APPROVED** [review]

**Review by EinsteinNjoroge — APPROVED** [review]

**Review by j-pendlebury — APPROVED** [review]
Looks good 👍

**Comment by jamesbrumpton** [comment]
lgtm

**Pull request merged**